src/main/java/customerproductorder/web/Controllers/CustomerController.java

```diff
@@ -0,0 +1,50 @@
+package customerproductorder.web.Controllers;
+
+import customerproductorder.models.Customer;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+import services.customer.CustomerServiceInterface;
+
+@Controller
+@ComponentScan("customerapplication.services.customer")
```

Comment:
This annotation should be on some Application Configuration class or in main application context xml. Otherwise it will be hard to find spring configuration.